File: RiotApiChallenge/WebApi/Models/Riot/Participant.cs
using Newtonsoft.Json;

namespace WebApi.Models.Riot
{
    public class Participant
    {
        [JsonIgnore]
        public int InternalParticipantId { get; set; }

        public int TeamId { get; set; }
        public int Spell1Id { get; set; }
        public int Spell2Id { get; set; }
        public int ChampionId { get; set; }
        public string HighestAchievedSeasonTier { get; set; }
    }
}

File: Android/GuessTheUrf/app/src/main/java/com/guesstheurf/guesstheurf/models/riot/Participant.java
package com.guesstheurf.guesstheurf.models.riot;


import android.os.Parcel;
import android.os.Parcelable;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Participant implements Parcelable {
    @JsonProperty("TeamId")
    private int teamId;

    @JsonProperty("Spell1Id")
    private int spell1Id;

    @JsonProperty("Spell2Id")
    private int spell2Id;

    @JsonProperty("ChampionId")
    private int championId;

    public int getTeamId() {
        return teamId;
    }

    public int getSpell1Id() {
        return spell1Id;
    }

    public int getSpell2Id() {
        return spell2Id;
    }

    public int getChampionId() {
        return championId;
    }

    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeInt(teamId);
        dest.writeInt(spell1Id);
        dest.writeInt(spell2Id);
        dest.writeInt(championId);
    }
}
